rdocs/00Index.html

```diff
@@ -0,0 +1,484 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
```

Comment:
OK, sure. To clarify though, that would be in the url, is that OK?

--- Original Message ---

From: "Shivaram Venkataraman" <notifications@github.com>
Sent: March 19, 2015 10:16 PM
To: "amplab-extras/SparkR-pkg" <SparkR-pkg@noreply.github.com>
Cc: "Felix Cheung" <felixcheung_m@hotmail.com>
Subject: Re: [SparkR-pkg] Add SparkR API doc to SparkR github page (#227)

> @@ -0,0 +1,484 @@
> +<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">

Lets name this directory rdocs-1.2 or rdocs-master to correspond to the github branch name

---
Reply to this email directly or view it on GitHub:
https://github.com/amplab-extras/SparkR-pkg/pull/227/files#r26819212